final_prep/README.md

```diff
@@ -26,30 +26,30 @@ Take your time as you go back and refactor your exercises from each section. We'
 
 ### Time Management Prep
 In Mod 0 you've learned about different techniques for managing your time at Turing. Please create a calendar for your **first 3 weeks of Mod 1**. Feel free to make your calendar fit your style, but we suggest that your calendar should include the following:
-- [ ] Standard M1 class schedule (see M1 calendar [here](https://backend.turing.io/module1/)
+- [ ] Standard M1 class schedule (see M1 calendar [here](https://backend.turing.io/module1/))
 - [ ] Study/Project work time
 - [ ] Health + Wellness
 
 When you are finished, add screenshots of your calendar so we can provide feedback if needed!
 
-- `Add Week 1 Screenshot Here`
-- `Add Week 2 Screenshot Here`
-- `Add Week 3 Screenshot Here`
+- ![alt text](https://user-images.githubusercontent.com/58535045/118335365-548c4280-b4c4-11eb-89fa-503334b5d877.png "Week 1")
+- ![alt text](https://user-images.githubusercontent.com/58535045/118335788-3ffc7a00-b4c5-11eb-9b62-46aa6be44691.png "Week 2")
+- ![alt text](https://user-images.githubusercontent.com/58535045/118336057-cca73800-b4c5-11eb-9cad-27f577701c3d.png "Week 3")
 
 ### Mentorship Prep
 Mentorship is an integral part of the Turing experience and will help jumpstart your technical career. In order to get your mentor relationship started on the right foot, please complete the following deliverables:
 - [ ] Complete the [Mentorship DTR Prep](https://gist.github.com/ericweissman/51965bdcbf42970d43d817818bfaef3c)
-  - [ ] Add link to your gist here: 
+  - [ ] Add link to your gist here: [Mentorship DTR](https://gist.github.com/khoifishpond/a1565e878a819f471b5979819c26afb3)
 
 ### Lesson Prep
 You've learned a lot about how to take strong notes during Mod 0. Show us your skills while you learn how to pre-teach content for your first lesson in Mod 1!
 - [ ] Complete the [Pre Teaching Practice exercise](https://gist.github.com/ericweissman/0036e8fe272c02bd6d4bb14f42fd2f79) gist
-  - [ ] Add a link to your gist here:
+  - [ ] Add a link to your gist here: [Pre Teaching Practice](https://gist.github.com/khoifishpond/fb21fac6f8808d9b17542b25f529fcfe)
```

Comment:
Nice- I like seeing ample space to add notes later if needed. This also helps make visuals/charts easier to read.